From 6fe7ec5a089e76aa53356d9bce6496f4927fbca5 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 1 Feb 2013 14:16:53 +0100
Subject: [PATCH 236/628] spi: xilinx-qps: Remove unused header

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 5007bdaf62af144c23bc013d7cc1ea595fc5cc97

The driver doesn't need/require this header.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/spi/spi-xilinx-qps.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/spi/spi-xilinx-qps.c b/drivers/spi/spi-xilinx-qps.c
index c0375f7..3ebbd63 100644
--- a/drivers/spi/spi-xilinx-qps.c
+++ b/drivers/spi/spi-xilinx-qps.c
@@ -29,7 +29,6 @@
 #include <linux/spi/spi.h>
 #include <linux/spinlock.h>
 #include <linux/workqueue.h>
-#include <linux/xilinx_devices.h>
 
 /*
  * Name of this driver
-- 
1.7.5.4

